tests/integration/targets/test_mysql_user/tasks/test_privs.yml

```diff
@@ -173,12 +173,62 @@
         state: present
       register: result
 
-    # FIXME: on mariadb >=10.5.2 there's always a change because the REPLICATION CLIENT privilege was renamed to BINLOG MONITOR
+    # FIXME: on mysql >=8 and mariadb >=10.5.2 there's always a change because the REPLICATION CLIENT privilege was renamed to BINLOG MONITOR
     - name: Assert that priv did not change
       assert:
         that:
           - result is not changed
-      when: install_type == 'mysql' or (install_type == 'mariadb' and mariadb_version is version('10.2', '=='))
+      when: (install_type == 'mysql' and mysql_version is version('8', '<')) or (install_type == 'mariadb' and mariadb_version is version('10.2', '=='))
```

Comment:
I'm trying to understand the issue.
The assertion also fails with MySQL 5.7.31 (Python 3.9, pymysql 0.9.4, ansible 2.13) because result.changed is true. Result is :
```json
{
    "result": {
        "changed": true,
        "failed": false,
        "msg": "Privileges updated: granted ['GRANT', 'USAGE'], revoked []",
        "password_changed": false,
        "user": "db_user2"
    }
}
```